tox.ini

```diff
@@ -31,6 +31,18 @@ commands =
     python --version
     pytest -p no:warnings --cov=baybe --durations=5 {posargs}
 
+[testenv:singlepointprecision,singlepointprecision-py{310,311,312}]
```

Comment:
as discussed lets remove CI and tox until al tests can be run in singlepoint precision

Reply:
Removed.